curriculum/challenges/english/25-front-end-development/quiz-debugging-javascript/66edd10913f078e7669eca81.md

```diff
@@ -83,15 +83,15 @@ Which console method can be used to display data as a table?
 
 #### --text--
 
-Which of the following errors does `try...catch` handle?
+Select the error that the `try...catch` statement is used to handle.
```

Comment:
I remember a discussion about preferring the interrogative form (with the final question mark). I cannot find it though.

Reply:
Yeah, I think that we should keep this as a question since it matches the other quizzes.